lib/vsc/mympirun/scoop/myscoop.py

```diff
@@ -337,27 +347,56 @@ def scoop_run(self):
             self.scoop_hosts.insert(origin_idx, self.scoop_hosts[origin_idx])
             self.scoop_size += 1
 
-        scoop_app_args = [[(nodename, len(list(group))) for nodename, group in itertools.groupby(self.scoop_hosts)],
-                          self.scoop_size,
-                          self.scoop_verbose,
-                          [self.scoop_python],
-                          self.scoop_broker,
-                          self.scoop_executable,
-                          self.scoop_args,
-                          self.scoop_tunnel,
-                          None,  # TODO args.log, deal with fancylogger later
-                          self.scoop_path,
-                          self.scoop_debug,
-                          self.scoop_nice,
-                          "other",  # TODO check utils.getEnv(),
-                          self.scoop_profile,
-                          self.scoop_pythonpath[0],
-                          # custom
-                          self.scoop_freeorigin,
-                          self.scoop_processcontrol,
-                          self.scoop_affinity,
-                          vars_to_pass,
-                          ]
+        if LooseVersion(SCOOP_VERSION) >= LooseVersion('0.7'):
```

Comment:
can you set the old list and insert / change things for version 0.7. this is way too much copy/paste